homeassistant/components/squeezebox/coordinator.py

```diff
@@ -70,3 +74,46 @@ def _prepare_status_data(self, data: dict) -> dict:
 
         _LOGGER.debug("Processed serverstatus %s=%s", self.lms.name, data)
         return data
+
+
+class SqueezeBoxPlayerUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class SqueezeBoxPlayerUpdateCoordinator(DataUpdateCoordinator[...]):
```
Can we add the response value of async update as type generic?